File: Code Chef/JavaScript/700-799/776 Helping Chef FLOW008.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => Number(strItem.trim()) );
    
    return intArray;
};

const printMsg = (num) => {
    const msg = (num < 10)
        ? 'Thanks for helping Chef!' : -1;
    console.log(msg);
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((num) => {
        printMsg(num);
    });
});

File: Code Chef/C Programming/700-799/776 Helping Chef FLOW008.c
#include <stdio.h>
int main(void)
{
	int t,n;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d",&n);
	    if(n>=10)
	    {
	        printf("-1\n");
	    }
	    else
	    {
	        printf("Thanks for helping Chef!\n");
	    }
	}
	return 0;
}